File: 23 mar 22/calculation.java
import java.lang.*;
import java.util.*;

class solve
{
	public int a,b;
	
	public solve(int x, int y)
	{
		a=x;
		b=y;
	}
	
	public void sum()
	{
		int ans;
		ans=a+b;
		System.out.println("Summetion is :" +ans);
	}

	public void sub()
	{
		int ans;
		ans=a-b;
		System.out.println("Subtraction is :" +ans);
	}

	public void div()
	{
		int ans ;
		ans=a/b;
		System.out.println("Division is :" +ans);
	}

	void mult()
	{
		int ans;
		ans=a*b;
		System.out.println("Multiplication is :" +ans);
	}
	
	
}

class calculation
{
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter First Number :");
		int ino1;
		ino1=sobj.nextInt();
		
		System.out.println("Enter Second Number :");
		int ino2;
		ino2=sobj.nextInt();
		
		solve obj = new solve(ino1,ino2);
		
		obj.sum();
		obj.sub();
		obj.div();
		obj.mult();
		
	}
}

File: 23 mar 22/calculation.cpp
#include<iostream>

using namespace std;

class calculation
{
	public:
	int a;
	int b;
	int ans;
	
	calculation(int ino1,int ino2)
	{
		a=ino1;
		b=ino2;
	}
	
	void sum()
	{
		int ans;
		ans=a+b;
		cout<<"Summetion is :"<<ans<<"\n";
	}

	void sub()
	{
		int ans;
		ans=a-b;
		cout<<"Subtraction is :"<<ans<<"\n";
	}

	void div()
	{
		int ans ;
		ans=a/b;
		cout<<"division is :"<<ans<<"\n";
	}

	void mult()
	{
		int ans;
		ans=a*b;
		cout<<"multiplication is :"<<ans<<"\n";
	}
};

int main()
{
	int x,y;
	cout<<"Enter first number"<<"\n";
	cin>>x;
	cout<<"Enter second number"<<"\n";
	cin>>y;
	
	calculation obj(x,y);
	obj.sum();
	obj.sub();
	obj.div();
	obj.mult();
	
	return 0;
}